### PR TITLE
rename data-cy selectors to data-testid to match playwright default locators

### PR DESCRIPTION
**Pull request by theobadha**
There were some remnants of elements that had the default locator for cypress `data-cy`
I renamed these to `data-testid` to match the default locator for playwright.

This PR fixes [this issue](https://github.com/layer5labs/kanvas-snapshot/issues/104)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@AmTryingMyBest 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

**Comment by leecalcote** [comment]
// @ianrwhitney

**Review by ianrwhitney — COMMENTED** [review]
would you be able to cleanup the whitespace changes? Also are these ids used anywhere? Any tests that will break by just chaning the id and not the test.

File: ui/components/MesherySettingsEnvButtons.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ const MesherySettingsEnvButtons = () => {
           marginRight: '2rem',
         }}
         disabled={!CAN(keys.ADD_CLUSTER.action, keys.ADD_CLUSTER.subject)}
-        data-cy="btnResetDatabase"
+        data-testid="btnResetDatabase"
```
Comment (ianrwhitney) [inline ui/components/MesherySettingsEnvButtons.js:213]: looks like we have a mix of camel case and kebab case. Any way we can align those?

Comment (AmTryingMyBest) [reply ui/components/MesherySettingsEnvButtons.js]: @ianrwhitney I made these changes. All names are now written in kebab case. :)

**Comment by AmTryingMyBest** [comment]
> would you be able to cleanup the whitespace changes? Also are these ids used anywhere? Any tests that will break by just chaning the id and not the test.

@ianrwhitney  I did the clean up for the whitespaces using `lint`. No live (e2e) tests are affected by these changes. 
This PR may be ready, but when updating I may have run some git commands incorrectly, since now within the PR there's over 5000+ files changed?  😕 
Let me know if I went about updating my branch wrongly or if I need to make any changes to or with my PR for this to be done correctly or better.
cc @leecalcote @jerensl

**Comment by leecalcote** [comment]
@AmTryingMyBest you'll want to resync your fork and this branch with the latest changes in this upstream repo. A description of this process can be found in CONTRIBUTING.MD.

**Pull request closed without merging**